master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml

```diff
@@ -320,7 +320,44 @@ spec:
             # kubernetes.default to the correct service clusterIP.
             - name: CONFIGURE_ETC_HOSTS
               value: "true"
+
 ---
+
+# This deployment turns off the old "policy-controller". It should remain at 0 replicas, and then
+# be removed entirely once the new kube-controllers deployment has been deployed above.
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: calico-policy-controller
+  namespace: kube-system
+  labels:
+    k8s-app: calico-policy
+spec:
+  # Turn this deployment off in favor of the kube-controllers Deployment above.
```

Comment:
"Deployment" --> "deployment"

Reply:
Not clear whether they should both be capitalized or both lower case... similar to the network policy vs NetworkPolicy discussion.  In this particular case we are referring to a Deployment object.

I'll make them all lowercase for now.